app/scripts/lib/app-start.js

```diff
@@ -160,19 +162,14 @@ function (
     },
 
     _trackWindowOnError: function () {
-      var self = this;
-      // if startup is okay we want to log future window.onerror events
-      window.onerror = function (message /*, url, lineNumber*/) {
-        var errMsg = 'null';
-
-        if (message) {
-          errMsg = message.toString().substring(0, Constants.ONERROR_MESSAGE_LIMIT);
-        }
+      var sentryEndpoint = 'http://__API_KEY__@' + this._window.location.host + '/metrics-errors';
```

Comment:
Will the "http://" cause any issues in stage/prod running on HTTPS?
Or can we use something protocol agnostic like:
```js
var sentryEndpoint = '//__API_KEY__@' + this._window.location.host + '/metrics-errors';
```